### ISSUE TITLE
GBDT原理与Sklearn源码分析-分类篇

### ISSUE THREAD

**Issue by xiang578** [xiang578/reading#155]
<b>GBDT&#21407;&#29702;&#19982;Sklearn&#28304;&#30721;&#20998;&#26512;-&#20998;&#31867;&#31687;</b><br>
&#25688;&#35201;&#65306; &#32487;&#19978;&#19968;&#31687;&#25991;&#31456;&#65292;&#20171;&#32461;&#23436;&#22238;&#24402;&#20219;&#21153;&#19979;&#30340;GBDT&#21518;&#65292;&#36825;&#31687;&#25991;&#31456;&#23558;&#20171;&#32461;&#22312;&#20998;&#31867;&#20219;&#21153;&#19979;&#30340;GBDT&#65292;&#22823;&#23478;&#23558;&#21487;&#20197;&#30475;&#21040;&#65292;&#23545;&#20110;&#22238;&#24402;&#21644;&#20998;&#31867;&#65292;&#20854;&#23454;GBDT&#36807;&#31243;&#31616;&#30452;&#23601;&#26159;&#19968;&#27169;&#19968;&#26679;&#30340;&#12290;&#22914;&#26524;&#35828;&#26368;&#22823;&#30340;&#19981;&#21516;&#30340;&#35805;&#65292;&#37027;&#23601;&#26159;&#22312;&#20110;&#30001;&#20110;loss function&#19981;&#21516;&#32780;&#24341;&#36215;&#30340;&#21021;<br>
<br>
Tags:<br>
<br>
via Pocket https://ift.tt/2Pe2iJl<br>
<br>
December 01, 2018 at 11:06PM
